stream-chat-android-compose/src/main/res/drawable/stream_compose_ic_person_remove.xml

```diff
@@ -0,0 +1,22 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="25dp"
+    android:height="24dp"
+    android:viewportWidth="25"
+    android:viewportHeight="24"
+    >
+    <path
+        android:fillColor="#000000"
```

Comment:
Need to decide, which is preferable for us:
- Black icons + required tint set externally with `iconTint`
- Icons with embedded color

Reply:
All icons should have an `iconTint` set externally, because we're using our ChatTheme.

That being said, for the purpose of the icon being visible here and so that we can inspect it, we put the color as black

Reply:
Ok, for view-based UI components we decided to embed tint so that the icons can be set externally by the client (can even be a png and not a vector drawable).